modules/maxquant/lfq/#main.nf#

```diff
@@ -0,0 +1,32 @@
+process MAXQUANT_LFQ {
```

Comment:
please remove this file

Reply:
still there though... please take a look at the "files changed" tab

Reply:
Now it should be gone